features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/modules/statistics/usage.jag

```diff
@@ -296,17 +321,26 @@ function getSubscriberCountByAPIs(providerName, isAllStatistics) {
 function getProviderAPIVersionUserUsage(providerName,apiName,version,serverUrl) {
     var usage = [];
     var usageout = [];
+
+    if(usageClient == null){
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/d7e56b1d85254369d07aab9572f13d4c2120df97